I18N.md

```diff
@@ -0,0 +1,55 @@
+## i18n
+
+See [i18n](link) for information on our internationalization tools and guidelines
+
+We use [react-i18next](https://react.i18next.com/) to translate messages in assisted-ui-lib. The
+i18n namespace must be declared in TRANSLATION_NAMESPACE property to avoid naming conflicts.
+
+- Environment variables:
+
+```
+  TRANSLATION_NAMESPACE: required, name of translation namespace
+```
+
+To test to standalone we need to put TRANSLATION_NAMESPACE=cim_namespace
+
+- We create the wrapper src/common/i18n/cim-18n that use the `useTranslation` hook with our
+  namespace as follows:
+
+```tsx
+import { useTranslation as useReactI18NextTranslation } from 'react-i18next';
+
+export function useTranslation() {
+  return useReactI18NextTranslation(`${process.env.TRANSLATION_NAMESPACE}`);
+}
+```
+
+- Use in our classes :
+
+```tsx
+import { useTranslation } from '../../i18n/cim-i18n';
+....
+ <Button key="submit" onClick={submitForm}>
+  {t('ai:Generate Discovery ISO')}
+</Button>
+```
+
+- File scripts/i18n-parser.config.js allow to generate the translation files automatically inside
+  root/locales folder:
+
+```js
+module.exports = {
+  contextSeparator: '_',
+  createOldCatalogs: false,
+  keySeparator: false,
+  locales: ['en'],
+  defaultNS: 'cim_namespace', // the default file for strings when using useTranslation, etc
+  namespaceSeparator: '~',
+  reactNamespace: false,
+  useKeysAsDefaultValue: true,
+  output: './locales/$LOCALE/translation.json',
+  sort: true,
+};
+```
```

Comment:
I dont think we need to have this part in the readme. Anybody can just look at `yarn i18n` script (which is mentioned below) and see what is going on .

Reply:
new proposal of i18n.md:
![image](https://user-images.githubusercontent.com/11390125/171004061-1fd0c95f-baeb-4719-8787-0fd57eeb180b.png)